src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractReferenceFinder.cs

```diff
@@ -564,6 +567,33 @@ static bool IsRelevantDocument(SyntaxTreeIndex syntaxTreeInfo)
             }
         }
 
+        protected Task<ImmutableArray<FinderLocation>> FindReferencesInImplicitObjectCreationExpressionAsync(
+            ISymbol symbol,
+            Document document,
+            SemanticModel semanticModel,
+            CancellationToken cancellationToken)
+        {
+            return FindReferencesInDocumentAsync(symbol, document, IsRelevantDocument, CollectMatchingReferences, cancellationToken);
+
+            static bool IsRelevantDocument(SyntaxTreeIndex syntaxTreeInfo)
+                => syntaxTreeInfo.ContainsImplicitObjectCreation;
+
+            void CollectMatchingReferences(ISymbol originalUnreducedSymbolDefinition, SyntaxNode node,
+                ISyntaxFactsService syntaxFacts, ISemanticFactsService semanticFacts, ArrayBuilder<FinderLocation> locations)
+            {
+                var constructor = semanticFacts.GetImplicitObjectCreationConstructor(semanticModel, node);
+
+                if (Matches(constructor, originalUnreducedSymbolDefinition))
+                {
+                    var location = node.GetFirstToken().GetLocation();
+                    var symbolUsageInfo = GetSymbolUsageInfo(node, semanticModel, syntaxFacts, semanticFacts, cancellationToken);
+
+                    locations.Add(new FinderLocation(node, new ReferenceLocation(
+                        document, alias: null, location, isImplicit: true, symbolUsageInfo, GetAdditionalFindUsagesProperties(node, semanticModel, syntaxFacts), CandidateReason.None)));
```

Comment:
does `GetAdditionalFindUsagesProperties(node, semanticModel, syntaxFacts)` work on these?

Reply:
Strangely, I could not find a test that is affected by commenting out logic in `GetAdditionalFindUsagesProperties(node, semanticModel, syntaxFacts)`.
It works though (tested manually) and also reviewed what compiler APIs it calls.

![image](https://user-images.githubusercontent.com/12466233/81513341-bb3cb600-92dc-11ea-80a1-0ca16e3df97a.png)

Reply:
👍